src/agent.rs

```diff
@@ -165,3 +179,38 @@ impl Agent {
         output.clone()
     }
 }
+
+fn check_cipher_blacklist(cipher: &str, shim: &str) -> bool {
+    let mut bl = fs::File::open("cipher_blacklist.json").unwrap();
```

Comment:
So the blacklist filename can't be changed?

Reply:
Right now it can't. Do you think it's important to make that possible?

Reply:
I moved the specification of the blacklist file name to the main function, so it's easier to find and change. Is that ok for now or should I make the effort to introduce a new command line parameter for this? I don't really expect that to be necessary, since the vlacklist is supposed to be pretty static once all currently failing ciphers are investigated.

Reply:
That's fine, maybe move the string to a constant so it's easier to find/change.